rubocop-rspec.yml

```diff
@@ -223,7 +224,7 @@ RSpec/VoidExpect:
 RSpec/Yield:
   Enabled: true
 
-RSpec/Capybara/FeatureMethods:
+RSpec/Dialect:
```

Comment:
Does `RSpec/Dialect` do anything on its own or do you need to set up `PreferredMethods`? (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Dialect)

Reply:
Done! It didn’t change the behavior, it seems :sweat_smile: